src/source/Rtcp/RtpRollingBuffer.c

```diff
@@ -33,9 +33,12 @@ STATUS addRtpPacket(PRollingBuffer pRollingBuffer, PRtpPacket pRtpPacket)
 {
     STATUS retStatus = STATUS_SUCCESS;
     PRtpPacket pRtpPacketCopy = NULL;
+    PBYTE rawPacketCopy = NULL;
     CHK(pRollingBuffer != NULL && pRtpPacket != NULL, STATUS_NULL_ARG);
 
-    CHK_STATUS(createRtpPacketFromBytes(pRtpPacket->pRawPacket, pRtpPacket->rawPacketLength, &pRtpPacketCopy));
+    rawPacketCopy = MEMALLOC(pRtpPacket->rawPacketLength);
```

Comment:
cast the allocation to PBYTE to avoid compiler warnings and check for the allocation not being NULL